index.html

```diff
@@ -251,33 +248,33 @@
 
                 return `
                     <h1>Your data</h1>
-                    <p>
+
```

Comment:
Ik heb de p hier weggehaald, omdat je geen div'jes in een paragraph element mag gebruiken. Elke browser gaat hier subtiel anders mee om, dus het veiligst is om de P weg te halen

Reply:
thanks!